content/en/docs/contributing/_index.md

```diff
@@ -36,13 +37,14 @@ To get invites you can subscribe to [our mailing list](https://groups.google.com
 We have 2 regular repeating meetings:
 
 * our daily stand-up meetings [on Google Meet](
-https://meet.google.com/eum-fyvt-xpa) at 10:30 London time every weekday 
-* our bi-weekly developer meetings [are on Google Meet](https://meet.google.com/abp-bwhk-wxc) 17:00 London time every 2 weeks (check calendar invites or [meeting notes](https://docs.google.com/document/d/1Tc5t6ylY9dhXAan1OjOoldeaoys1Yh4Ir710ATfBa5U)). 
+https://meet.google.com/eum-fyvt-xpa) at 10:30 London time every weekday
+* our bi-weekly developer meetings [are on Google Meet](https://meet.google.com/abp-bwhk-wxc) 17:00 London time every 2 weeks (check calendar invites or [meeting notes](https://docs.google.com/document/d/1Tc5t6ylY9dhXAan1OjOoldeaoys1Yh4Ir710ATfBa5U)).
```

Comment:
Nit that I couldn't resist: 'bi-weekly...every 2 weeks' <- some redundancy of information

Reply:
> Nit that I couldn't resist: 'bi-weekly...every 2 weeks' <- some redundancy of information

Yep agreed, no need for redundancy there. I'll remove.

> would it make sense to call it something like 'coding conventions' rather than style guide

I think that's a much better name for the document, yeah. I'll change it.